aio-libs/aiohttp-security
# Default handler decorator for authentication

## What these changes does?
- Removed authorized_userid() from AbstractAuthorizationPolicy #3 
- Added default handler decorator #2 
## How to test your changes?
- refer to [session_auth demo code](https://github.com/shicky/aiohttp_security/tree/default_decorator/demo/session_auth)
  to see it in action.
- make test passes
## Related issue number
#3 #2
## Checklist
- [ O ] Code is written and well
- [ O ] Tests for the changes are provided
- [ O ] Documentation reflects the changes
